package.json

```diff
@@ -14,8 +14,7 @@
   "dependencies": {
     "css-vendor": "^0.3.1",
     "debug": "^2.1.3",
-    "lodash.throttle": "^3.0.3",
-    "object.assign": "^4.0.1"
```

Comment:
I did not consider the intermediate compilation. Hopefully, webpack 2 and Babel will be able to use modules written in ES6 without [hack](https://github.com/kriasoft/react-starter-kit/blob/master/tools/webpack.config.js#L162). Ok I will remove it from the pull request.